contents/docs/integrate/_snippets/install-react-native.mdx

```diff
@@ -39,111 +41,47 @@ export function MyApp() {
         </PostHogProvider>
     )
 }
+```
 
-// Now you can simply access PostHog elsewhere in the app like so:
+Then you can access PostHog using the `usePostHog()` hook:
 
+```react-native
 const MyComponent = () => {
     const posthog = usePostHog()
 
     useEffect(() => {
-        posthog.capture("mycomponent_loaded", { foo: "bar" })
-    }, [])
+        if (posthog) {
+            posthog.capture("event_name")
+        }
+    }, [posthog])
 }
 ```
 
 #### Without the PosthogProvider
 
+If you prefer not to use the provider, you can initialize PostHog in it's own file and import the instance from there:
 
-```react-native
-// posthog.ts
+```react-native file=posthog.ts
 import PostHog from 'posthog-react-native'
 
 export const posthog = new PostHog('<ph_project_api_key>', {
   // usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'  
   host: '<ph_client_api_host>' // TIP: host is optional if you use https://us.i.posthog.com
 })
+```
 
+Then you can access PostHog by importing your instance:
 
-// app.ts
+```react-native
 import { posthog } from './posthog'
 
 export function MyApp1() {
     useEffect(async () => {
-        posthog.capture('myapp1_loaded')
+        if (posthog) {
+            posthog.capture('event_name')
+        }
     }, [])
 
     return <View>Your app code</View>
 }
-
-
-// You can even use this instance with the PostHogProvider
-export function MyApp2() {
-  return <PostHogProvider client={posthog}>{/* Your app code */}</PostHogProvider>
-}
```

Comment:
Some people prefer this because they can hold and share the `posthog` instance, any reason to remove this?

Reply:
will add back!